_plugins/rosindex_generator.rb

```diff
@@ -1082,7 +1082,7 @@ def generate(site)
                       end
 
                       # get the tracks file
-                      ['master','bloom'].each do |branch_name|
+                      ['master','bloom', 'main'].each do |branch_name|
```

Comment:
Is this change related?

Reply:
This is the minor drive-by change that I mentioned in the PR intro. While investigating the code to understand why some repos were or were not being included, I came across a ros gbp repo that used "main" instead of "master" as its branch name. Unfortunately I don't see any documentation in my notes about which one this was. I'll remove this change.

Reply:
Ahh, thanks I hadn't looked deep enough in. I believe that bloom still assumes `master` as the name I think. This is something that could be extended here for future compatibility.